_posts/en/newsletters/2020-10-07-newsletter.md

```diff
@@ -0,0 +1,198 @@
+name: 2020-10-07-newsletter
+slug: 2020-10-07-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter describes a revised proposal for a generic
+message signing protocol and summarizes a discussion about merging
+code for taproot support into Bitcoin Core.  Also included are our
+regular sections describing releases, release candidates, and notable
+changes to popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Alternative to BIP322 generic signmessage:** the existing [BIP322][]
+  proposes a [generic message signing protocol][topic generic
+  signmessage] for Bitcoin that would allow signing messages for any
+  Bitcoin address (script) even if it used multisig or advanced
+  features.  This includes the ability to sign for all types of bech32
+  addresses, a feature that has not yet been standardized even several years
+  after the introduction of that type of address and its subsequent
+  widespread adoption (see [Newsletter #54][news54 bech32 signing]).
+
+    This week, BIP322 author Karl-Johan Alm [posted][alm signmessage] to
+    the Bitcoin-Dev mailing list an alternative proposal for message
+    signing that would use a technique called virtual transactions.  The
+    first virtual transaction would be deliberately invalid by
+    attempting to spend from a non-existent previous transaction (one
+    who's txid was all zeroes).  This first transaction pays the address
+    (script) the user wants to sign for and contains a hash commitment
+    to the desired message.  A second transaction spends the output of
+    the first transaction---if the signatures and other data for that
+    spend could be a valid transaction, then the message is considered
+    signed (although the second virtual transaction still can't be
+    included onchain because it spends from an invalid previous
+    transaction).
+
+    The advantage to using virtual transactions, which were also adopted
+    in the [BIP325][] specification for [signet][topic signet] (see
+    [Newsletter #109][news109 signet bip]), is that they may work with
+    existing software that's willing to sign arbitrary transactions,
+    including those in [PSBT][topic psbt] format.  The revised
+    specification also allows the virtual transaction to contain inputs
+    that reference specific UTXOs, allowing users to (arguably) prove
+    control over those funds, similar to [BIP127][] proof of reserves.
+
+    Alm is seeking feedback on the alternative proposal, including
+    whether or not it should replace BIP322 or be opened as a separate
+    BIP.
+
+- **Brief discussion about merging taproot in Bitcoin Core:** this
+  week's Bitcoin Core Developer Meeting included a [brief discussion][meeting
+  taproot] about whether the main taproot implementation ([PR
+  #19953][bitcoin core #19953]) should be merged in the next couple
+  weeks before the version 0.21 code is scheduled to be branched off,
```

Comment:
maybe s/branch-off/feature freeze/ (I'm guessing you simplified it for the reader, but since FF is mentioned in the next paragraph it may be fine to say it here)